cuda/nn/organized_neighbor_search.hpp

```diff
@@ -246,8 +246,8 @@ namespace pcl
       if ((int)nearestNeighbors.size () == k_arg)
       {
         double squared_radius;
-        unsigned int pointCountRadiusSearch;
-        unsigned int pointCountCircleSearch;
+        //unsigned int pointCountRadiusSearch;
+        //unsigned int pointCountCircleSearch;
```

Comment:
Why not removing code, wich is not required anymore? We have git in case someone want restore it ;-)

Reply:
I thought it should be kept reference for this line:
`if (1){//(pointCountCircleSearch<pointCountRadiusSearch) {`

I'll remove it.

Reply:
Now that you mention it, we should also remove the if-else block, since it is `if(1)`

Reply:
Done.

Reply:
Good catch with `if(1)` 👍